### PR TITLE
Update pull_request_template.md with hopefully clearer language

### PR DESCRIPTION
**Pull request by josueetcom**
## Motivation and Context

I'm not sure if the template instructions are clear enough so I've tried to make them clearer and slightly simpler.

### CODE REVIEW

File: .github/pull_request_template.md
Comment (edwingl10) [inline .github/pull_request_template.md:37]: I think this is what is making the test fail.

**Review by edwingl10 — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md
Comment (josueetcom) [reply .github/pull_request_template.md:37]: Dang yeah you're right. Hmm, I need to make a two line change to get this to work. This PR is low priority so I think I'm going to work on that PR that auto-approves/keeps approval for small changes after an approval.

**Review by Jobhdez — APPROVED** [review]
Looks good

**Review by Jobhdez — DISMISSED** [review]
uses clear language -- it is clear as it is but maybe the person who is contributing should add comments to the code? It will make the code easier to understand. maybe the person who is contributing should also make changes to the documentation if necessary. the contributor should not assume the maintainer is going to update the README for instance.

**Comment by josueetcom** [comment]
> uses clear language -- it is clear as it is but maybe the person who is contributing should add comments to the code? It will make the code easier to understand. maybe the person who is contributing should also make changes to the documentation if necessary. the contributor should not assume the maintainer is going to update the README for instance.

I believe when it refers to "documentation" it's likely referring to documentation that library packages provide. As we're not a library package besides README.md I don't know what else might need regular updating. Whether or not to update the README can probably be determined by the reviewer.

**Comment by Jobhdez** [comment]
> > uses clear language -- it is clear as it is but maybe the person who is contributing should add comments to the code? It will make the code easier to understand. maybe the person who is contributing should also make changes to the documentation if necessary. the contributor should not assume the maintainer is going to update the README for instance.
> 
> I believe when it refers to "documentation" it's likely referring to documentation that library packages provide. As we're not a library package besides README.md I don't know what else might need regular updating. Whether or not to update the README can probably be determined by the reviewer.
> 
> Regarding code in comments, I generally expect code to be self-documenting (e.g. `name` is some `string` with the value of a name) as it's hard to keep comments up to date with code. Comments become dated really quickly. Though comments that I think are valuable are those that explain why something strange/complex is done. My general expectation is that a reviewer with knowledge of React/JS that doesn't understand why something is done or thinks something's not clear would ask for a comment be added.

ok sounds good! that makes sense comments are not necessarily needed.

**Review by gonzalo1330 — APPROVED** [review]

**Review by edwingl10 — APPROVED** [review]

**Pull request merged**
